src/main/java/org/takes/rq/RqLive.java

```diff
@@ -63,7 +63,7 @@ public RqLive(final InputStream input) throws IOException {
      */
     @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     private static Request parse(final InputStream input) throws IOException {
-        final Collection<String> head = new LinkedList<String>();
+        final List<String> head = new LinkedList<String>();
```

Comment:
@lautarobock Makes sense, thanks. Btw, please address me in the previous comment with @carlosmiranda. All messages should start with someone's name.